Pascal/Pascal.sublime-syntax

```diff
@@ -47,7 +47,7 @@ contexts:
             0: punctuation.definition.comment.pascal
           push:
```

Comment:
I'm not sure what the `\G` is for. Maybe not needed at all, but as I said, not very familiar with how this all works